js/plugins/google-cloud/tests/metrics_test.ts

```diff
@@ -260,7 +252,8 @@ describe('GoogleCloudMetrics', () => {
     );
   });
 
-  it('writes action metrics for a failed action', async () => {
+  // SKIPPED -- we don't allow defining arbitrary actions anymore....
+  it.skip('writes action failure metrics', async () => {
```

Comment:
What's the eventual plan w/ the skipped tests?

Reply:
Filed #1071 to track, but we need to decide whether these are even worth testing since generic actions are not possible to create anymore... maybe can be switched to use some other type of action.